server.go

```diff
@@ -245,6 +245,23 @@ func checkDefaultSecurityTests(configAPI *apiContext.APIConfig) error {
 		return err
 	}
 
+	retirejsQuery := map[string]interface{}{"name": "retirejs"}
+	retirejs, err := analysis.FindOneDBSecurityTest(retirejsQuery)
+	if err == mgo.ErrNotFound {
+		// As Retire JS securityTest is not set into MongoDB, HuskyCI will insert it.
```

Comment:
Would mind fixing this typo? `RetireJS`

Reply:
Sure can! Fixed this typo mistake in 1dc1c00